docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+toml
```

Comment:
For what purposes do we need **requirements.txt**? Can't we just generate in on-the-fly?

For example using example from [here ](https://docs.readthedocs.io/en/latest/config-file/v1.html#python-extra-requirements). To add additional section to the **pyproject.toml** file, then parse it in **setup.py** and add additional extra reqs that could be used by **readthedocs.yml**